libslab/app-resizer.c

```diff
@@ -223,11 +211,8 @@ app_resizer_size_allocate (GtkWidget * widget, GtkAllocation * allocation)
 	GtkRequisition other_requisiton;
 	gtk_widget_get_preferred_size (GTK_WIDGET (resizer->cached_tables_list->data), &other_requisiton, NULL);
 
-	useable_area =
-		allocation->width - (child_requisition.width -
-		other_requisiton.width);
 	new_num_cols =
-		relayout_tables_if_needed (APP_RESIZER (resizer), useable_area,
+		relayout_tables_if_needed (APP_RESIZER (resizer), allocation->width,
```

Comment:
Because we removed ```gtk_table_resize (table, (int) rows, columns);```,  ```child_requisition.width``` should not be used,```child_requisition.width``` will always change with the increase of components.